### ISSUE TITLE
DOC: Major overhaul of scipy.constants documentation?

### ISSUE THREAD

**Issue by jakobjakobson13** [scipy/scipy#11173]
Hi,

I personally find the [scipy.constants](https://docs.scipy.org/doc/scipy/reference/constants.html) a not very concise collection of units and constants.
If you don't have any objections, I would write up a draft following the SI-brochure. This could either mean a restructured and extended documentation leaving all the existing information on the page or a slimmed down variant that drops the units that are not part of the SI conventions.

Regards

**Comment by rgommers** [comment]
Agreed, thanks @jakobjakobson13. In particular the huge list of physical constants is a bit jarring. Slimming down that page and moving things to subpages with some logic that follows the SI brochure seems like a good idea.

**Comment by jakobjakobson13** [comment]
I'm closing this one as it's too disruptive.
